.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
```

Comment:
This was copied from https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/dependabot.yml